src/assets/pages/blog.json

```
@@ -1,10 +1,20 @@
+      "banner": "https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/blog/post0058/banner-post58.png",
```

```suggestion
      "banner": "https://querido-diario-static.nyc3.cdn.digitaloceanspaces.com/blog/post0059/banner-post59.png",
```

E precisa adicionar a imagem da Digital Ocean...